book/layout.md

```diff
@@ -620,30 +575,29 @@ class Browser:
         self.max_y = self.document.height - HEIGHT
 ```
 
-Then, when the user scrolls down, don't let them scroll past the
-bottom of the page:
+When the user scrolls down, don't let them scroll past the bottom of
+the page:
 
 ``` {.python}
 def scrolldown(self, e):
-    self.scroll = self.scroll + SCROLL_STEP
-    self.scroll = min(self.scroll, self.max_y)
-    self.scroll = max(0, self.scroll)
+    self.scroll = min(self.scroll + SCROLL_STEP, self.max_y)
     self.render()
 ```
 
-Make sure those `max` and `min` calls happen in the right order!
+Thank you tree-based layout! In fact, as we'll see in the next two
```

Comment:
Why thank you? Is there a convenient feature you're referring to?

If, not, perhaps "Well that's tree-based layout!"